ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -229,7 +243,20 @@ def __get_table_entity(
         )
         table_fqn = _get_formmated_table_name(table_fqn)
         table_entity = self.metadata.get_by_name(Table, fqdn=table_fqn)
-        return table_entity
+        if table_entity:
+            return [table_entity]
+        es_result = self.metadata.search_entities_using_es(
+            service_name=self.service_name,
+            table_obj={
+                "database": database_name,
+                "database_schema": database_schema,
+                "name": table_name,
+            },
+            search_index="table_search_index",
+        )
+        if es_result:
+            return es_result
+        return []
```

Comment:
might be cleaner to return `None` instead of an empty list. we'll then need to check if we got some results. Also pls add the type hints to the function. This one should then return `Optional[something]`.
If is then the same as saying `return es_result or None`